### ISSUE TITLE
TFLite not giving output

### ISSUE THREAD

**Issue by SanthoshRajendiran** [hyperconnect/MMNet#2]
We tried out your model. The output quality is good. We referred your paper, wherein it mentions that you got good result in tflite level. We tried out your pb file output, and the output is coming good. When we tried out the tflite conversion, the tflite is created and output is not coming properly. There are huge changes in the weights between the frozen graph convolution operation and tflite convolution operation when the tflite conversion happens. We are using the fused batch norm approach. Need ur help

**Comment by shurain-hpcnt** [comment]
Have you performed fake quantization for quantization-aware training? Refer to [Quantization and training of neural networks for efficient integer-arithmetic-only inference](https://arxiv.org/abs/1712.05877).

**Comment by Nerdyvedi** [comment]
@SanthoshRajendiran . Were you able to get the right results from the tflite model ?
